source/dem/dem.cc

```diff
@@ -143,60 +143,6 @@ DEMSolver<dim>::DEMSolver(DEMSolverParameters<dim> dem_parameters)
     });
 #endif
 
-
```

Comment:
Shouldn't you call the is_contact_search_step() here?

Reply:
No because this is the constructor of the class. What used to be done here was just establishing the right function pointer. This is why I wanted to remove that function pointer logic, kinda created a two-step logic abstraction that we don't need

Reply:
Ok. Got it. I thought I have deleted this comment actually :)